doc/admin/deployment.md

```diff
@@ -365,36 +365,39 @@ The --enable-recovery is required for fault tolerance to guarantee that
 the fault of one server does not cause the others to be stopped.
 
 The --allow-run-as-root option can be added to the command line to
-allow the daos_server to run with root privileges on each storage
+allow the `daos_server` to run with root privileges on each storage
 nodes (for example when needing to perform privileged tasks relating
 to storage format). See the orterun(1) man page for additional options.
 
 #### Systemd Integration
 
 DAOS Server can be started as a systemd service. The DAOS Server
 unit file is installed in the correct location when installing from RPMs.
+The DAOS Server will be run as `daos-server` user which will be created
+during RPM install.
+
 If you wish to use systemd with a development build, you must copy the service
-file from utils/systemd to /usr/lib/systemd/system. Once the file is copied
-modify the ExecStart line to point to your in tree daos_server binary.
+file from utils/systemd to `/usr/lib/systemd/system`. Once the file is copied
+modify the ExecStart line to point to your in tree `daos_server` binary.
 
-Once the service file is installed you can start daos_server
+Once the service file is installed you can start `daos_server`
 with the following commands:
 
 ```bash
-$ systemctl enable daos-server
-$ systemctl start daos-server
+$ systemctl enable daos_server
+$ systemctl start daos_server
```

Comment:
I believe it's most correct to put the type of unit you are working on on the unit name:

```suggestion
$ systemctl enable daos_server.service
$ systemctl start daos_server.service
```